code/modules/food_and_drinks/food/snacks_pie.dm

```diff
@@ -28,33 +28,51 @@
 	tastes = list("pie" = 1)
 	foodtype = GRAIN | DAIRY | SUGAR
 	var/stunning = TRUE
+	var/experience_given = 10
```

Comment:
I feel like you should multiply this amount by at least 10, if not 100, or else no one will ever approach the clown's rouundstart skill at pie-ing people (due to the relative rarity of cream pies).

You could compensate for this proposed change by making it so that pie-ing someone who still has crea pie residue on their face give 1/10 or 1/100 as much experience, to discourage farming/bullying.

Reply:
Don't forget to modify the experience_given value for syndicate pies as well,

Reply:
100 pies to legendary?

Reply:
For a clown to get to legendary or for a non-clown to get to legendary?